packages/core/src/lib/jdk/JdkHelper.ts

```diff
@@ -110,9 +110,9 @@ export class JdkHelper {
     try {
       const releaseFilePath = join(javaHome, 'release');
       const file = await fsPromises.readFile(releaseFilePath, 'utf-8');
-      if (file.indexOf('JAVA_VERSION="11.0') < 0) { // Checks if the jdk's version is 11 as needed
+      if (file.indexOf('JAVA_VERSION="17.0') < 0) { // Checks if the jdk's version is 11 as needed
```

Comment:
comment for if statement should be:
```diff
-// Checks if the jdk's version is 11 as needed
+// Checks if the jdk's version is 17 as needed
```

Reply:
Good catch! I've updated for that in a new commit.